packages/p2p/src/components/orders/orders.jsx

```diff
@@ -6,23 +6,31 @@ import OrderInfo    from './order-info';
 import OrderTable   from './order-table/order-table.jsx';
 import './orders.scss';
 
-const Orders = () => {
-    const buy_order = new OrderInfo();
-    const sell_order = new OrderInfo();
-    sell_order.type = 'sell';
-    sell_order.order_id = 'SELL123';
-    sell_order.counterparty = 'Ronald McDonald';
-
+const Orders = ({ params }) => {
+    // const buy_order = new OrderInfo();
+    // const sell_order = new OrderInfo();
+    // sell_order.type = 'sell';
+    // sell_order.order_id = 'SELL123';
+    // sell_order.counterparty = 'Ronald McDonald';
     const [order_details, setDetails] = React.useState(null);
     // TODO: [p2p-replace-with-api] - remove these dev toggle once data fetch works
-    const showDetails = (order) => setDetails(order);
+    const showDetails = (order) => {
+        setDetails(order);
+    };
```

Comment:
Can we just change this to `const showDetails = setDetails;`? I do still believe the renaming is necessary though.